debian/control

```diff
@@ -26,6 +26,7 @@ Depends:
     miniupnpc (>= 1.9),
     gobject-introspection,
     gir1.2-gtk-3.0,
+    gir1.2-notify-0.7,
```

Comment:
Should we add this to README.md as well? I'm not sure if this is a Debian and derivative specific nomenclature or if this package is more generally known by this name in the wild.

Reply:
Probably a good idea! I know nothing about this package, but it looks awfully specific.

Reply:
At least on Manjaro/Arch, everything seems to be bundled in one "libnotify" package.

Reply:
I'll add it anyways for safe measure.